web/client/src/components/infrastructure/elementTypes.ts

```diff
@@ -1,23 +1,41 @@
-export const elementTypeLabels = {
-    'bumper': 'Bumper',
-    'border': 'Border',
-    'track_end': 'Track End',
-    'simple_switch': 'Switch',
-    'as': 'Approach Signal',
-    'ms': 'Main Signal',
-    'ps': 'Protection Signal',
-    'eotd': 'End of Train Detector',
-    'spl': 'Speed Limit',
-    'tunnel': 'Tunnel',
-    'hlt': 'Halt',
-    'rtcp': 'RTCP',
-    'km_jump': 'KM Jump',
-    'line_switch': 'Line Switch',
-    'slope': 'Slope',
-    'cross': 'Cross',
-    'ctc': 'CTC',
-    'station': 'Station'
+export const ElementTypes = {
+    BUMPER: 'bumper',
+    BORDER: 'border',
+    TRACK_END: 'track_end',
+    SWITCH: 'simple_switch',
+    APPROACH_SIGNAL: 'as',
+    MAIN_SIGNAL: 'ms',
+    PROTECTION_SIGNAL: 'ps',
+    END_OF_TRAIN_DETECTOR: 'eotd',
+    SPEED_LIMIT: 'spl',
+    TUNNEL: 'tunnel',
+    HALT: 'hlt',
+    RTCP: 'rtcp',
+    KM_JUMP: 'km_jump',
+    LINE_SWITCH: 'line_switch',
+    SLOPE: 'slope',
+    CROSS: 'cross',
+    CTC: 'ctc',
+    STATION: 'station',
 };
```

Comment:
Extract a array of all elements so you dont have to call Object.values all the time